src/api/app/assets/stylesheets/webui2/webui2.css.scss

```diff
@@ -47,10 +47,6 @@ html {
     overflow-y: scroll !important;
 }
 
-.dataTable {
```

Comment:
It didn't break any table when it was added. Removing it breaks the rest of the tables now.

Reply:
I don't think we should "patch" all data tables. If you need that for some datatables, add a class to these.

Reply:
@Ana06 and I discussed offline that the best approach is to not enable it by default but add a new class for it because it is extremely hard to debug that the tables are not behaving like stated in the documentation. @Ana06 will take care of a PR and I will drop the commit when it is merged.

Reply:
@ChrisBr here it is: https://github.com/openSUSE/open-build-service/pull/6435 :wink:

Reply:
@ChrisBr rebase, it is now in master